lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -1,130 +1,174 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
+
 # Copyright: Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: postgresql_ext
-short_description: Add or remove PostgreSQL extensions from a database.
+short_description: Add or remove PostgreSQL extensions from a database
 description:
-   - Add or remove PostgreSQL extensions from a database.
+- Add or remove PostgreSQL extensions from a database.
 version_added: "1.9"
 options:
   name:
     description:
-      - name of the extension to add or remove
+    - Name of the extension to add or remove.
     required: true
+    type: str
   db:
     description:
-      - name of the database to add or remove the extension to/from
+    - Name of the database to add or remove the extension to/from.
     required: true
+    type: str
+    aliases:
+    - login_db
   schema:
     description:
-      - name of the schema to add the extension to
+    - Name of the schema to add the extension to.
     version_added: "2.8"
```

Comment:
Minor thing, though since you are in these PRs anyway:
```suggestion
    version_added: '2.8'
```

Reply:
we discussed similar things yesterday ;) Ok, I don't mind about single or dowble quotes, I'll change it